interview-questions.md

```diff
@@ -7,44 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: Migrations can be used to update the database! I need to research the rest. 
 
-  Researched answer:
+  Researched answer: A developer can perform a migration to create a relationship between two databases using a foreign key. The foreign key would be named has_many :students and I think it would be on the Cohort model.
```

Comment:
Please review this section of the syllabus 
https://github.com/learn-academy-2022-alpha/Syllabus/blob/main/rails/associations.md